### PR TITLE
PAYARA-2341 repackage JCIP annotations with OSGi headers

### PR DESCRIPTION
**Pull request by arjantijms**
PAYARA-2341 creates a JCIP jar with the OSGi headers in MANIFEST.MF, but does not yet add this as a dependency to Payara.

### CODE REVIEW

**Comment by arjantijms** [comment]
jenkins test please

**Comment by smillidge** [comment]
Can you add an entry in https://github.com/payara/Payara/blob/master/appserver/packager/legal/src/main/resources/glassfish/legal/PAYARA-3RD-PARTY-LICENSE.txt if this is a new 3rd party dependency being packaged into Payara

File: appserver/packager/external/jcip/pom.xml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                        <id>unpack</id>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.github.stephenc.jcip</groupId>
+                                    <artifactId>jcip-annotations</artifactId>
+                                    <version>1.0-1</version>
```
Comment (Pandrex247) [inline appserver/packager/external/jcip/pom.xml]: Please define and use a version property from appserver/pom.xml

**Comment by arjantijms** [comment]
jenkins test please

**Pull request merged**